qemu/tests/timerdevice_boot.py

```diff
@@ -173,14 +135,15 @@ def verify_guest_clock_source(session, expected):
                       " host's. Host time: '%s', guest time:"
                       " '%s'" % (host_time, guest_time))
 
-        get_hw_time_cmd = params.get("get_hw_time_cmd")
-        if get_hw_time_cmd:
+        if params["os_type"] == "linux":
             error_context.context(
-                "Check the hardware time on guest and host", logging.info)
-            host_time = process.system_output(get_hw_time_cmd, shell=True)
-            guest_time = session.cmd(get_hw_time_cmd)
-            drift = abs(float(host_time) - float(guest_time))
+                "Check the hardware time on guest", logging.info)
+            guest_hwtime = get_hwtime(session)
+            drift = abs(float(host_time) - float(guest_hwtime))
```

Comment:
I think you should get these two time together, instead of using the previous host_time directly, which makes the comparison not accurate.

Reply:
@xiangchunfu Would you help confirm is that okay of this modification, to compare guest's hardware time with host's system time instead of it's hardware time.